bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -43,13 +46,20 @@ def _bool_from_json(value, field):
         return value.lower() in ['t', 'true', '1']
 
 
-def _datetime_from_json(value, field):
+def _timestamp_from_json(value, field):
     """Coerce 'value' to a datetime, if set or not nullable."""
     if _not_null(value, field):
         # value will be a float in seconds, to microsecond precision, in UTC.
         return _datetime_from_microseconds(1e6 * float(value))
 
 
+def _datetime_from_json(value, field):
+    """Coerce 'value' to a datetime, if set or not nullable."""
+    if _not_null(value, field):
+        # value will be a string, in YYYY-MM-DDTHH:MM:SS form.
+        return datetime.datetime.strptime(value, '%Y-%m-%dT%H:%M:%S')
```

Comment:
@tseaver Don't we already have this template in `google.cloud._helpers`? If not, please add it there.